### DISCUSSION TITLE
Need advice

### DISCUSSION THREAD

**Discussion by Vidmantask1** [arkypita/LaserGRBL#1626] [Q&A]
![PROBLEMA 1](https://user-images.githubusercontent.com/92754097/137792914-43dbb698-10dd-40f0-8cee-ffcd7b719867.jpg)
![PROBLEMA](https://user-images.githubusercontent.com/92754097/137792967-b0c854bf-e6a6-4da4-be74-2d820986e963.PNG)
Hello,
Case 1. The check function burns.
Case 2. The laser burns between the ornaments.

**Comment by gmmanonymus111** [comment]
Sorry for the late reply,
My first guess is your laser driver reads TTL signal in inverse. When it should be off (or low power), it actually burns on high power. That's why it burns  on preview, and that's why it leaves a line while traveling.
Read more about the issue [here](https://github.com/gnea/grbl/issues/22)

**Reply by Vidmantask1** [reply]
Hi, crashed after updating app.

**Reply by gmmanonymus111** [reply]
What crashed? Any error messages? Without sufficient info, no-one can help.

**Reply by Vidmantask1** [reply]
Hello. Version v4.6.0 works better. Only problem 1 remains.
